src/HelperPluginManager.php

```diff
@@ -143,41 +147,78 @@ class HelperPluginManager extends AbstractPluginManager
      * @var array
      */
     protected $factories = [
-        'flashmessenger'                  => Helper\Service\FlashMessengerFactory::class,
-        'identity'                        => Helper\Service\IdentityFactory::class,
+        Helper\FlashMessenger::class      => Helper\Service\FlashMessengerFactory::class,
+        'zendviewhelperflashmessenger'    => Helper\Service\FlashMessengerFactory::class,
+
+        Helper\Identity::class            => Helper\Service\IdentityFactory::class,
+        'zendviewhelperidentity'          => Helper\Service\IdentityFactory::class,
+
         Helper\BasePath::class            => InvokableFactory::class,
+        'zendviewhelperbasepath'          => InvokableFactory::class,
         Helper\Cycle::class               => InvokableFactory::class,
+        'zendviewhelpercycle'             => InvokableFactory::class,
         Helper\DeclareVars::class         => InvokableFactory::class,
+        'zendviewhelperdeclarevars'       => InvokableFactory::class,
         Helper\Doctype::class             => InvokableFactory::class, // overridden by a factory in ViewHelperManagerFactory
+        'zendviewhelperdoctype'           => InvokableFactory::class,
         Helper\EscapeHtml::class          => InvokableFactory::class,
+        'zendviewhelperescapehtml'        => InvokableFactory::class,
```

Comment:
Separate the FQCN and normalized names, and group them by type. This will allow us to remove the normalized versions more easily at a later date. (I can do this on merge if needed.)